src/components/pagination/_pagination.scss

```diff
@@ -219,13 +222,18 @@ $css--helpers: true;
     .#{$prefix}--select-input {
       width: auto;
       min-width: auto;
-      height: auto;
-      padding: 0 1.5rem 0 0.5rem;
-      // TODO: @include type-style('body-short-01');
-      font-size: rem(12px);
-      font-weight: regular;
-      line-height: rem(18px);
-      letter-spacing: 0.16px;
+      height: 100%;
+      padding: 0 2.5rem 0 $spacing-md;
+      margin-right: -0.65rem;
+      @include breakpoint($pagination-breakpoint) {
+        padding: 0 1.5rem 0 0.5rem;
+        margin-right: 0;
+      }
+      @include type-style('body-short-01');
```

Comment:
Style nit: should we move the `type-style` block to the top? Only rationale would be that we can call mixins at the top-level (unless the order of properties matter 👍)